lib/src/collection/multimap.dart

```diff
@@ -186,12 +186,12 @@ abstract class _BaseMultimap<K, V, C extends Iterable<V>>
   }
 
   void add(K key, V value) {
-    _map.putIfAbsent(key, _create);
+    _map.putIfAbsent(key, () => _create() as C);
```

Comment:
This allocates a closure for every add. Let's see if we can avoid it. Can `_create` return `C`?